scripts/performance-helper.psm1

```diff
@@ -561,30 +561,42 @@ function Get-TestResult($Results, $Matcher) {
     }
 }
 
-function Get-LatestCommitHash([string]$Branch) {
+function Get-LatestCommitHashes([string]$Branch) {
     $Uri = "https://raw.githubusercontent.com/microsoft/msquic/performance/data/$Branch/commits.json"
     Write-LogAndDebug "Requesting: $Uri"
     try {
         $AllCommits = Invoke-RestMethod -SkipHttpErrorCheck -Uri $Uri -Method 'GET' -ContentType "application/json"
         Write-LogAndDebug "Result: $AllCommits"
-        $LatestResult = ($AllCommits | Sort-Object -Property Date -Descending)[0]
-        Write-LogAndDebug "Latest Commit: $LatestResult"
-    return $LatestResult.CommitHash
+        if ($AllCommits.Count -eq 0) {
+            return ""
+        }
+        $SortedList = $AllCommits | Sort-Object -Property Date -Descending
+        if ($SortedList.Count -lt 5) {
```

Comment:
Can we make this `5` a global var or something, so it's easily adjusted (possibly) in the future? Maybe to 10 instead.